lore/developers/contributing/designs/personal-cloud-accounts.md

```diff
@@ -0,0 +1,32 @@
+---
+name: personal cloud accounts
+layout: default
+---
+
+# Personal Cloud Accounts
+
+## Problem Statement
+
+We now have our own Bluemix Private Cloud, with cloud_admin rights to it. This cloud is primarily intended to be used to run our BonnyCI control plane and to provide nodepool capacity for running tests. Additionally, we can make use of this cloud for personal development environments. In order to keep track of personal projects and to ensure they are created and managed in a sane way, we need to introduce some controls and automation around their creation.
+
+## Overview
+
+Each BonnyCI developer is entitled to a personal cloud account and matching project, with an appropriate role. A reasonable set of quotas will be enforced. Accounts and projects will be created via automation.
+
+### Accounts Projects and Roles
+
+An account is a set of credentials to log into OpenStack via keystone. An account by itself is not very useful, as it requires a role within a project. A single account may have multiple roles across mutiple projects. For our cloud, each developer is entitled to an account with a matching project, and a role of _member_ within the project.
```

Comment:
`project_admin` role allows creating new accounts (though they can only be granted roles on the given project). Tracking which of these accounts "belong" to a given project and should be cleaned up should a project get removed would be difficult. I feel it would be best to keep account creation to automation. 

The help doc that is linked to describes the capabilities of the roles.